content/master/software/upgrade.md

```diff
@@ -47,7 +47,7 @@ Crossplane uses any new default behaviors unless they're changed in the `helm
 upgrade` command.
 
 For example, in v1.15.0 Crossplane changed the default image registry from 
```

Comment:
should this be `v1.20.0` and `xpkg.upbound.io --> xpkg.crossplane.io`?

Reply:
fixed